openstack/grafana/templates/postgres-service.yaml

```diff
@@ -13,3 +13,28 @@ spec:
     component: grafana-postgres
   ports:
     - port: 5432 
+    
+{{- if .Values.metrics.enabled }}
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: grafana-postgres-backup-metrics
+  labels:
+    app: grafana-postgres
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "9188"
+spec:
+  clusterIP: None
+  ports:
+    - name: backup-metrics
+      port: 9188
+      protocol: TCP
+  selector:
+    app: grafana-postgres
```

Comment:
I think the label selector is wrong. https://github.com/sapcc/helm-charts/blob/eaf6c35a79d22f13b246358df958e9e5c06a44a7/openstack/grafana/templates/postgres-deployment.yaml#L18-L19